src/sentry/api/serializers/models/userreport.py

```diff
@@ -29,15 +29,18 @@ def get_attrs(self, item_list, user):
     def serialize(self, obj, attrs, user):
         # TODO(dcramer): add in various context from the event
         # context == user / http / extra interfaces
+        name = obj.name or obj.email
+        email = obj.email
+        if attrs["event_user"]:
+            event_user = attrs["event_user"]
+            if isinstance(event_user, dict):
+                name = name or event_user.get("name")
```

Comment:
Looks like `event_user.get("username")` would also need to be considered to preserve the previous behaviour.

Reply:
The EventUser serializer does the coalesce into name.